### ISSUE TITLE
Scheduling Transactional Emails

### ISSUE THREAD

**Issue by sverraest** [mailjet/mailjet-apiv3-php#128]
I've looked at the blogpost from May 2017

https://www.mailjet.com/blog/news/meet-our-brand-new-send-api/

and the current documentation 

https://dev.mailjet.com/guides/#send-api-v3-1

and I can't seem to find any docs for Mailjet Pipe or scheduling emails in the future to be delivered. Is there any update on this?

**Comment by kmanchikov** [comment]
Hi Simon, 

For now the priority of this release has been moved a bit back in the roadmap, since we had been working on fixing issues with a higher priority and releasing more requested features than this one. I won't be able to provide you with an exact ETA for now, but surely we would revert back to Email Schedualing a bit further in time. 

Best, 
Krassimir

**Comment by LrntL** [comment]
For my project, I chose to manage transactional emails in a database table.

A cronjob takes care of sending them with or without delay according to the date of sending (for each email to send).

- Prepare recipient
- Prepare content
- Define a sending date
- Store the whole thing
- Let the cronjob do his job

With that I can also check if there is an error during the sending.
I did not find other ways to do that.
